Non-Forge builds loading as Forge mods

## Some information
Operating system: any
Java version: any
Minecraft version: 1.15.2+, Forge
Baritone version: 1.5.3+, non-Forge
Forge mods (if used): any

## Exception, error or logs
<details><summary>log (copied from <a href='https://github.com/cabaletta/baritone/issues/2115#issuecomment-716225108'>#2115 (comment)</a>)</summary><pre>---- Minecraft Crash Report ----
// This doesn't make any sense!

Time: 26/10/20 9:45 AM
Description: Unexpected error

org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:223) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinTransformationHandler.processClass(MixinTransformationHandler.java:121) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.launch.MixinLaunchPlugin.processClass(MixinLaunchPlugin.java:131) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:154) ~[modlauncher-7.0.1.jar:7.0.1+78+master.e9771d8] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:85) ~[modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:119) ~[modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader$DelegatedClassLoader.findClass(TransformingClassLoader.java:256) ~[modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:135) ~[modlauncher-7.0.1.jar:?] {re:classloading}
	at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:97) ~[modlauncher-7.0.1.jar:?] {re:classloading}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_74] {}
	at net.minecraft.client.multiplayer.ClientChunkProvider.<init>(ClientChunkProvider.java:50) ~[?:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.world.ClientWorld.<init>(ClientWorld.java:117) ~[?:?] {re:classloading,xf:OptiFine:default,xf:fml:xaerobetterpvp:xaero_clientworldclass}
	at net.minecraft.client.network.play.ClientPlayNetHandler.func_147282_a(ClientPlayNetHandler.java:375) ~[?:?] {re:classloading,xf:fml:xaerobetterpvp:xaero_clientplaynethandler_cleanup,xf:fml:xaerobetterpvp:xaero_clientplaynethandler_handlejoingame,xf:fml:xaerobetterpvp:xaero_clientplaynethandler_handlemultiblockchange,xf:fml:xaerobetterpvp:xaero_clientplaynethandler_handlechunkdata,xf:fml:xaerobetterpvp:xaero_clientplaynethandler_handleblockchange,xf:fml:xaerobetterpvp:xaero_clientplaynethandler_handlespawnpoint,xf:fml:xaerobetterpvp:xaero_clientplaynethandlerclass,pl:runtimedistcleaner:A}
	at net.minecraft.network.play.server.SJoinGamePacket.func_148833_a(SourceFile:110) ~[?:?] {re:classloading}
	at net.minecraft.network.play.server.SJoinGamePacket.func_148833_a(SourceFile:18) ~[?:?] {re:classloading}
	at net.minecraft.network.PacketThreadUtil.lambda$checkThreadAndEnqueue$0(PacketThreadUtil.java:34) ~[?:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(ThreadTaskExecutor.java:172) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
	at net.minecraft.util.concurrent.RecursiveEventLoop.func_213166_h(SourceFile:23) ~[?:?] {re:classloading}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(ThreadTaskExecutor.java:134) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213160_bf(ThreadTaskExecutor.java:115) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:924) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:584) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) ~[1.16.3-forge-34.1.23.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_74] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_74] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_74] {}
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_74] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) ~[forge-1.16.3-34.1.23.jar:34.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81) [modlauncher-7.0.1.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65) [modlauncher-7.0.1.jar:?] {}
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [mixins.baritone.json:MixinChunkArray] from phase [DEFAULT] in config [mixins.baritone.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:642) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:594) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:356) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	... 33 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow method b in mixins.baritone.json:MixinChunkArray was not located in the target class net.minecraft.client.multiplayer.ClientChunkProvider$ChunkArray. Using refmap mixins.baritone.refmap.json
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachSpecialMethod(MixinPreProcessorStandard.java:429) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachShadowMethod(MixinPreProcessorStandard.java:402) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachMethods(MixinPreProcessorStandard.java:330) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:289) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:267) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1271) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:287) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
	... 33 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.16.3
	Minecraft Version ID: 1.16.3
	Operating System: Mac OS X (x86_64) version 10.15.3
	Java Version: 1.8.0_74, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1477712224 bytes (1409 MB) / 2147483648 bytes (2048 MB) up to 2147483648 bytes (2048 MB)
	CPUs: 4
	JVM Flags: 8 total; -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	ModLauncher: 7.0.1+78+master.e9771d8
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.2.jar mixin PLUGINSERVICE 
		/eventbus-3.0.3-service.jar eventbus PLUGINSERVICE 
		/forge-1.16.3-34.1.23.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.3-34.1.23.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-2.2.0-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.3-34.1.23.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.16.3-34.1.23.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.2.jar mixin TRANSFORMATIONSERVICE 
		/OptiFine_1.16.3_HD_U_G3.jar OptiFine TRANSFORMATIONSERVICE 
		/forge-1.16.3-34.1.23.jar fml TRANSFORMATIONSERVICE 
	FML: 34.1
	Forge: net.minecraftforge:34.1.23
	FML Language Providers: 
		javafml@34.1
		minecraft@1
	Mod List: 
		forge-1.16.3-34.1.23-client.jar Minecraft {minecraft@1.16.3 DONE}
		baritone-standalone-1.6.1.jar Baritone {baritoe@1.6.1 DONE}
		BetterPvP_20.26.0_Forge_1.16.3.jar Better PVP Mod {xaerobetterpvp@20.26.0 DONE}
		forge-1.16.3-34.1.23-universal.jar Forge {forge@34.1.23 DONE}
		worldedit-forge-mc1.16.3-7.2.0-rc-01.jar WorldEdit {worldedit@7.2.0-rc-01+81f978e DONE}
	Crash Report UUID: 6fe6026a-d0e6-43e9-a42a-5796b5ad6e9e
	Launched Version: 1.16.3-forge-34.1.23
	Backend library: LWJGL version 3.2.1 build 12
	Backend API: Intel(R) HD Graphics 6000 GL version 2.1 INTEL-14.4.23, Intel Inc.
	GL Caps: Using framebuffer using ARB_framebuffer_object extension
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	GPU Warnings: version: 2.1
	Graphics mode: fancy
	Resource Packs: vanilla, file/Transparant UI
	Current Language: English (US)
	CPU: 4x Intel(R) Core(TM) i5-5250U CPU @ 1.60GHz
	OptiFine Version: OptiFine_1.16.3_HD_U_G3
	OptiFine Build: 20200916-082411
	Render Distance Chunks: 16
	Mipmaps: 4
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 2.1 INTEL-14.4.23
	OpenGlRenderer: Intel(R) HD Graphics 6000
	OpenGlVendor: Intel Inc.
	CpuCount: 4</pre></details>

## How to reproduce
1. Put wrong jar (standalone or api instead of standalone-forge or api-forge) of a release that has forge jars into mods folder
2. Start game
3. Join world
4. crash

## What should happen
1. Put wrong jar (standalone or api instead of standalone-forge or api-forge) of a release that has forge jars into mods folder
2. Start game
3. Be told that baritone-\*-\*.jar is not a valid mod

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [x] I have included logs, exceptions and / or steps to reproduce the issue.
- [x] I have not used any OwO's or UwU's in this issue, excluding the two on this line.

--- a/build.gradle
+++ b/build.gradle
@@ -117,6 +117,12 @@ javadoc {
 
 jar {
     from sourceSets.launch.output, sourceSets.api.output
+
+    if (!getProject().hasProperty("baritone.forge_build")) {
+        exclude "**/BaritoneForgeModXD.class"
+        exclude "**/mods.toml"
+    }
+
     preserveFileTimestamps = false
     reproducibleFileOrder = true
 

